Website/src/rendering/src/components/Navigation/HeaderContent.tsx

```diff
@@ -14,13 +15,26 @@ export type HeaderContentProps = HeaderProps & {
 const HeaderContent = (props: HeaderContentProps): JSX.Element => {
   const router = useRouter();
   const { sitecoreContext } = useSitecoreContext();
+  const [languageLabels, setLanguageLabels] = useState([]);
 
   const languageNames = new Intl.DisplayNames(['en'], {
     type: 'language',
   });
 
   const languageList = sitecoreContext['Languages'] as NodeJS.Dict<string | string>[];
 
+  useEffect(() => {
+    // Fix for React hydration error in MS Edge because of the display name mismatch for 'fr-CA'
```

Comment:
Does it mean we'd need it for every language we use at the site? Any chance to fix it in a generic way for all languages?

Reply:
It would have been only for 'fr-CA' because of the different value returned by MS Edge's implementation of Intl.DisplayNames(), but eventually it seems that there's an even simpler approach